includes/payment-methods/class-wc-stripe-upe-payment-method.php

```diff
@@ -549,6 +562,60 @@ public function should_show_save_option() {
 		return $this->is_reusable() && $this->is_saved_cards_enabled();
 	}
 
+	/**
+	 * Returns the payment method's limits per currency.
+	 *
+	 * @return int[][][]
+	 */
+	public function get_limits_per_currency(): array {
+		return $this->limits_per_currency;
+	}
+
+	/**
+	 * Returns the current order amount (from the "pay for order" page or from the current cart).
+	 *
+	 * @return float|int|string
+	 */
+	public function get_current_order_amount() {
+		if ( is_wc_endpoint_url( 'order-pay' ) && isset( $_GET['key'] ) ) {
+			$order = wc_get_order( absint( get_query_var( 'order-pay' ) ) );
+			return $order->get_total( '' );
+		} elseif ( WC()->cart ) {
+			return WC()->cart->get_total( '' );
+		}
+		return 0;
+	}
+
+	/**
+	 * Determines if the payment method is inside the currency limits.
+	 *
+	 * @return bool True if the payment method is inside the currency limits, false otherwise.
```

Comment:
```suggestion
	 * @param  string $current_store_currency The store's currency.
	 * @return bool True if the payment method is inside the currency limits, false otherwise.
```

Just adding a `@param` description comment. :)